x/scheduler/keeper/keeper.go

```diff
@@ -21,80 +22,78 @@ import (
 )
 
 type Keeper struct {
-	cdc        codec.BinaryCodec
-	storeKey   storetypes.StoreKey
-	memKey     storetypes.StoreKey
-	paramstore paramtypes.Subspace
+	cdc      codec.BinaryCodec
+	storeKey store.KVStoreService
+	memKey   storetypes.StoreKey
```

Comment:
You should use same type for both `storeKey` and `memKey`.

Reply:
Updated and pushed